packages/landing/src/api.ts

```diff
@@ -0,0 +1,27 @@
+import * as ulid from 'ulid';
+
+const hasLocalStorage = typeof localStorage != 'undefined';
+const ONE_DAY_MS = 24 * 60 * 60 * 1000;
+const API_EXPIRE_TIME = ONE_DAY_MS; // TODO make this longer
+
+function newApiKey(): string {
+    const newKey = 'c' + ulid.ulid().toLowerCase();
+    if (hasLocalStorage) localStorage.setItem('api-key', newKey);
+    return newKey;
+}
+
+export function getApiKey(): string {
+    if (!hasLocalStorage) return newApiKey();
+    const apiKey = localStorage.getItem('api-key');
+
+    if (apiKey == null) return newApiKey();
+    if (!apiKey.startsWith('c')) return newApiKey();
+
+    try {
+        const keyCreatedAt = ulid.decodeTime(apiKey.slice(1).toUpperCase());
+        if (Date.now() - API_EXPIRE_TIME < keyCreatedAt) return apiKey;
+        return newApiKey();
+    } catch (e) {
+        return newApiKey();
+    }
```

Comment:
Why the try catch need here? Please add comment to code

Reply:
You dont want this ever to error, and we call into a third party library.

if a error occurs just generate a new key doesnt really matter about making more keys

Reply:
also protects against if the apikey is not a ulid